GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
+  - Implementation of the task would conflict with the project direction or [its values](https://biomejs.dev/internals/philosophy/).
+  - Implementation of the task is (currently) technically infeasible.
+  - The suggested pledge is deemed inappropriate for the amount of work involved.
+- The person making the pledge acknowledges their pledge does not come with a guarantee of work done.
+- A task is only completed when a Biome maintainer merges the pull request that closes the task. The person trying to complete the task is responsible for addressing review comments to make sure the work meets the quality standards of the Biome project.
+- There is a 6 month expiration on pledges, as managed by Polar.sh.
+- Terms and conditions of Polar.sh apply to all pledges.
+  - For the FAQ, please see: https://docs.polar.sh/maintainers/faq/
+  - For the legal terms, please see: https://polar.sh/legal/terms
+- If you are receiving money as part of a bounty, please consider whether you need to pay taxes in your country.
+
+### Project-Funded Bounties
+
+In some cases, Biome funds bounties on its own issues. For project-funded bounties, the same conditions apply as for community-funded bounties. But given that these issues are funded by money from our sponsors, we have additional restrictions in place:
+
+- Implementation of the task funded by Biome must advance [our roadmap](#roadmap) or help with essential infrastructure.
+  - The description of the task must include a clear rationale for why this task is eligible for funding.
+- Each Biome-funded task will be either assigned directly to a contributor, or assigned a label to indicate who is eligible for picking up that task:
+  - `F-Anyone`: Anyone is free to pick up this task. This task should be of low complexity and has a maximum bounty of $50 (USD).
+  - `F-Maintainers`: Any Biome maintainer is free to pick up this task. This task may be of medium complexity and has a maximum bounty of $200 (USD).
+  - `F-Core-Contributors`: Only core contributors are eligible for picking this task. There is no upper limit to the complexity, but the maximum bounty is $1000 (USD).
+- For Biome-funded tasks assigned to core contributors, we ask the contributor to take the money directly from our Open Collective account, instead of using Polar.sh. This will help us avoid paying commission over the larger bounties.
+
+Additionally, there is a strict process for assigning project-funded bounties:
+
+- To request Biome funding for a task, a request with a clear rationale must be made in the `Community > #bounties` channel on Discord.
+  - Merely commenting on a GitHub issue is not sufficient, to make sure project members have a better chance at monitoring requests.
+- If the request is approved by at least 2 maintainers and not vetoed by any core contributor within 7 days, then a core contributor can assign the pledge to the issue and assign the requester or a corresponding label to it.
```

Comment:
The community-funded bounties were decided by the core contributors, but these "bounties" are decided by maintainers too. Is this intentional or a mistake? This discrepancy should be clarified.

Also, I don't think Maintainers should be entitled to decide on funding matters, only the core team should be.

Reply:
Good catch, that's indeed a mistake. I'll just simplify it to "If no Core contributor raises objections within 7 days" and drop the part about maintainer approval.